### ISSUE TITLE
Feature Suggestion - (e)TICKER on all websites

### ISSUE THREAD

**Issue by nickmudry** [EddieRingle/p3_eav#6]
Wouldn't it be great to be able to click on any ticker that is posted in the form of (e)TICKER on any website and have it bring you to their ticker page? 

Just a suggestion and let me know what you think.

**Comment by EddieRingle** [comment]
Yeah, I've wanted to do this as well. Should be out in an upcoming release.
